svsim/src/main/scala/Workspace.scala

```diff
@@ -243,9 +243,11 @@ final class Workspace(
       l()
       l("extern \"C\" {")
       l(" svScope setScopeToTestBench();")
+      for ((port, index) <- ports) {
+      l(CreateFunctionForPort.createGetBitWidth(port.name))
+      }
       for ((port, index) <- ports.filter(_._1.isGettable)) {
       l(CreateFunctionForPort.createGetBits(port.name))
```

Comment:
```suggestion
      for ((port, index) <- ports) {
      l(CreateFunctionForPort.createGetBitWidth(port.name))
        if (port.isGettable) {
      l(CreateFunctionForPort.createGetBits(port.name))
        }
```
Stylistically, I think these would be better in a single loop (it also would keep the functions for a given port next to each other).

Reply:
Fused a setter and getter generation into a single loop.